SCClassLibrary/Common/GUI/tools/ServerMeter.sc

```diff
@@ -263,4 +263,13 @@ ServerMeter {
 	isClosed {
 		^window.isClosed
 	}
+
+	position_ { |point|
+		point = point.asPoint;
+		window.bounds_(window.bounds.left_(point.x).top_(point.y)).front
+	}
+
+	position {
+		^("Position(left:" ++ window.bounds.left ++ ", bottom:" ++ (window.bounds.bottom - window.bounds.height) ++ ")")
```

Comment:
If you'd like to create a getter method, best to return the actual values, as a Point.

Returning formatted text isn't a typical practice for getters, and the way this is formatted creates the illusion of a Position class, which doesn't exist.